### DISCUSSION TITLE
Are the docs outdated for engine (MODEL)

### DISCUSSION THREAD

**Discussion by Muhammad-I-Hasan** [visgl/luma.gl#2130] [Q&A]
Just taking a look at the docs for model and I've noticed some inconsistences within the code and the documentation, it states that Model.draw() takes in a few parameters here: 
model.draw({
  renderPass,
  moduleSettings = null,
  uniforms = {},
  attributes = {},
  samplers = {},
  parameters = {},
  settings,
  vertexArray = null,
  transformFeedback = null
});
But taking a look at the actual implementation the draw function only takes in a RenderPass as a parameter. My goal is to pass in certain uniforms to the draw function but i can no longer do that in the new version, and the setUniforms function is now deprecated.
It also states to use the model.setBindings() instead, but the documentation for this doesnt exist on Model, and on RenderPipeline it is simply a title and nothing else, could someone point me in the right direction as my goal is to pass in certain uniforms before drawing or if someone could example how to convert uniforms to uniform buffers, thanks

**Comment by ibgreen** [comment]
@Muhammad-I-Hasan  You are correct, the Model docs do indeed need a refresher. This is probably listed as a bullet in one of the release trackers, but a lot of other work has been taking priority.

And if there are some obvious things you feel you could address a PR would be most welcome.
